source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -87,6 +88,7 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::RequestHeaderMap& headers,
   if (verifier == nullptr) {
     onComplete(Status::Ok);
   } else {
+    original_uri_ = Http::Utility::buildOriginalUri(headers, 256);
```

Comment:
Where is 256 coming from?  Should we define it somewhere?

Reply:
256 seemed to be a good length limit for URIs in the headers. There are no actual defined limits for how long headers can be, effectively though, many servers will limit the total amount of data headers can use to 8KiB or 16KiB. I'm up for discussion on how long we should allow the generated URI to be. And then yes, I think it's a good idea define the value rather than leaving it as a raw number.

Reply:
It is fine to just const define it.

Reply:
Added